### DISCUSSION TITLE
Can we rename the project with lowercase letters ?

### DISCUSSION THREAD

**Discussion by nazimboudeffa** [detronetdip/E-commerce#10] [Ideas]
Hi,

I just want to ask if we can rename the project in something like **multivendor-e-commerce** or **multivendor-e-commerce-delivery** ?

A general name but with lowercase characters

Thank you

**Comment by detronetdip** [comment]
@nazimboudeffa actually it's a good idea but I have to check will it be ok or not to change the name.

**Reply by nazimboudeffa** [reply]
I understand, it may cause problems on the forks, but it's just an idea, no worries ;)
